hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/sink/ITTestDataStreamWrite.java

```diff
@@ -159,10 +159,14 @@ public void testWriteCopyOnWriteWithChainedTransformer() throws Exception {
   }
 
   @ParameterizedTest
-  @ValueSource(strings = {"BUCKET", "FLINK_STATE"})
+  @ValueSource(strings = {"BUCKET", "FLINK_STATE", "CONSISTENT_BUCKET"})
   public void testWriteMergeOnReadWithCompaction(String indexType) throws Exception {
     Configuration conf = TestConfigurations.getDefaultConf(tempFile.toURI().toString());
     conf.setString(FlinkOptions.INDEX_TYPE, indexType);
+    if (indexType.equals("CONSISTENT_BUCKET")) {
+      conf.setString(FlinkOptions.INDEX_TYPE, "BUCKET");
+      conf.setString(FlinkOptions.BUCKET_INDEX_ENGINE_TYPE, "CONSISTENT_HASHING");
+    }
```

Comment:
Can we write a separate test instead of modifying the existing one and also test the COW table.